supervised/loglizer/benchmarks/HDFS_bechmark.py

```
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import sys
+sys.path.append('../')
+import pandas as pd
+from loglizer.models import *
```

Ruff (0.8.2)

`from loglizer.models import *` used; unable to detect undefined names

(F403)